v3/src/components/axis/components/axis.scss

```diff
@@ -1,60 +1,64 @@
-.axis {
-  background-color: white;
+.graph-plot {
+  .axis {
+    background-color: white;
 
-  axis-title text {
-    fill: blue;
+    line {
+      stroke: #DDDDDD;
+    }
   }
-
-  line {
-    stroke: #DDDDDD;
+  .axis-click-cue {
+    fill: #999999 !important;
+    font-size: 12px !important;
+    font-family: Montserrat-Regular, sans-serif !important;
   }
-}
 
-.dragRect {
-  fill: #e38e71;
-  fill-opacity: 0.01;
-  cursor: grab;
-
-  &.dragging {
-    cursor: grabbing !important;
+  .grid .tick line {
+    stroke: lightgrey;
+    stroke-opacity: 0.7;
+    shape-rendering: crispEdges;
+    pointer-events: none;
   }
 
-  &.h-translate {
-    cursor: grab;
+  .zero .tick line {
+    stroke: #444444;
+    stroke-opacity: 0.8;
+    shape-rendering: crispEdges;
+    pointer-events: none;
   }
+}
+  // We can't put dragRect inside axis for some reason
+  .dragRect {
```

Comment:
Note that before my recent `strictFunctionTypes` PR, the `.dragRect` class was applied to both the individual drag rects _and_ their parent/wrapper, which could be related to the comment. I renamed the parent class to `.dragRectWrapper` so you could try embedding it inside `.axis` again.

Reply:
Not attempting now